index.html

```diff
@@ -176,6 +175,31 @@ <h5 class="categories-title">Skills</h5>
         </div>
     </div>
 </div>
+<form class="contact_forms" action="index.html" method="post">
+    <p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <ul class="form_field">
+        <li class="form_name">
+            <input type="text" id="first_name" name="first_name" class="input" placeholder="Full Name" maxlength="6" required>
```

Comment:
- [ ] As requested by the previous reviewer, the `maxlength` should be 500 and not 6. Please correct that. 👍

Reply:
yeah, good, it was a mistake, but you couldn't do it here, even on slack will be better and I come to do it here after you aprove, so now request an extra review, only because of one mistake 👎, that is no good my dear

Reply:
and moreover, it's not 500 rather 30